packages/pwa/app/components/_app/index.jsx

```diff
@@ -68,8 +70,11 @@ const App = (props) => {
     // Used to conditionally render header/footer for checkout page
     const isCheckout = /\/checkout$/.test(location?.pathname)
 
+    const l10nConfig = getL10nConfig(`${location?.pathname}${location.search}`)
```

Comment:
Again, if we have the site already, do we need to explicitly have another way to just get the l10n config?